src/development/coding-naming-conventions.md

```diff
@@ -111,7 +111,7 @@ Different case standards should be applied depending on usage:
 
 * `Natural` - Using natural language standard, with spaces to separate words and capitals for initial letter and proper nouns
 * `kebab-case` - All lower case with dashes to separate words
-* `CamelCase` - Initial letters in words are uppercased, with no separators
+* `CamelCase` - Initial letters in words are uppercased, with no separators. First letter may be upper or lower.
```

Comment:
Further info on camelCase at https://www.techtarget.com/whatis/definition/CamelCase#:~:text=What%20is%20CamelCase%3F,humps%20on%20a%20camel's%20back.
I always use first character lower case - but if you can uppercase it I suppose consistency is the key. Java tends to be lowerCamelCase.

Reply:
It depends on the circumstance @LouisRoberts. Class names are `CamelCase`, method names are `camelCase`